dandischema/models.py

```diff
@@ -807,6 +807,12 @@ class StandardsType(BaseType):
     identifier="RRID:SCR_016124",
 ).model_dump(mode="json", exclude_none=True)
 
+ome_ngff_standard = StandardsType(
+    name="OME/NGFF Standard",
+    # Seems no dedicated RRID, only for OME itself?
+    # identifier="RRID:SCR_???",
```

Comment:
@joshmoore - any thoughts here on identifier for the ome/ngff standard? or could we mint an RRID for it?

Reply:
:wave: @satra:

 * No identifier (I know of) in use at the moment.
 * Does it need to be an RRID?
 * Closest I know of is https://fairsharing.org/FAIRsharing.9af712 (https://doi.org/10.25504/FAIRsharing.9af712)

Reply:
thanks @joshmoore - doesn't have to be RRID, we can use any persistent identifier. we can use the doi.

Reply:
Let me run it by a few folks Monday AM, and I'll get back to you.

Reply:
Sorry for the wait, @satra. No other identifiers from our side. If the DOI works for the moment, then :+1: (How hard would this be to change in the future?)

Reply:
hyphen is fine. we can change it in the future. will just need to encode the change and the mapping.

Reply:
Sounds good.

Reply:
Thank you gentleman, I added DOI identifier.